docs/madr/decisions/033-deprecate-standalone.md

```diff
@@ -0,0 +1,84 @@
+# Deprecate standalone mode
+
+* Status: accepted
+
+## Context and Problem Statement
+
+We want to simplify our deployments model to only have Zone CP and Global CP.
+The goal is not to get rid of standalone altogether, but to change the naming.
+
+## Decision Drivers
+
+* Simplify UX for (new) users. Simpler docs. Fewer things to wrap your head around.
+* Provide better migration path from standalone to multizone, as standalone won't exist.
+
+## Considered Options
+
+* Deprecate standalone
+
+## Decision Outcome
+
+Chosen option: "Deprecate standalone".
+
+## Pros and Cons of the Options
+
+### Deprecate standalone
+
+If you want to run the "standalone" you can execute the following command
+```
+KUMA_MODE=zone kuma-cp run 
+```
+We will change the default `KUMA_MODE` value to `zone`. So in the end, you would run this the same way as before which is
```

Comment:
Do we need to provide default name of the zone, or it isn't required when no kds connection.

Reply:
Ideally, we would require it, but that would require action for existing users. That's why it's required only when we have KDS connection.